src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12887,44 +12992,68 @@ void func_8084FF7C(Player* this) {
     }
 }
 
-void func_8085002C(Player* this) {
-    s32 pad;
-    s16 sp2A;
-    s16 sp28;
-    s16 sp26;
+/**
+ * Updates the Bunny Hood's floppy ears' rotation and velocity.
+ *
+ * Acts as discretised versions (forward Euler method) of two uncoupled forced, damped harmonic oscillators
+ * \f[
+ *     \ddot{\theta} = - \frac{1}{8} \dot{theta} - \frac{1}{4} \theta + F(t) ,
+ * \f]
+ * with \f$ \lvert \dot{theta} \rvert \f$ clamped to \f$ [-6000, 6000] \f$. The force is determined by Player's velocity
+ * in its shape coordinate frame, although it is twice as large in the forward/backward direction as in the sideways
+ * ones:
+ * \f[
+ *     F_x(t) = -200 s (10 + r_x) \cos(\phi) \frac{1}{4} , \\
+ *     F_y(t) = 100 s (10 + r_y) \sin(\phi) \frac{1}{4} ,
+ * \f]
+ * where \f$ \phi \f$ is the horizontal angle between forwards and the velocity vector, and \f$ r_x,r_y \f$ are random
+ * numbers in \f$ [-1,1] \f$ that are picked every frame.
```

Comment:
regarding "too much documentation", I think most comments you added in the code are fine, but still not a fan of the latex stuff in comments. We removed these from the matrix stuff in previous prs, and dont think peoples stance on this has changed since